examples/rock_paper_scissors.py

```diff
@@ -33,56 +36,103 @@ def combine(self, other):
         return self.__class__(choice([self.preference, other.preference]))
 
 
-def evaluation_func(player_1, player_2):
+class RockPaperScissorsLizardSpockPlayer(RockPaperScissorsPlayer):
```

Comment:
note for later, but this might be a great candidate for the problems part of the library.